playbooks/continuous_delivery/image_promotion.adoc

```diff
@@ -119,3 +119,19 @@ docker push <promoted_docker_registry>/<ose_project>/<ose_app>
 docker rmi <origin_docker_registry>/<ose_project>/<ose_app>
 docker rmi <promoted_docker_registry>/<ose_project>/<ose_app>
 ----
+
+
+=== Promotion Process with `skopeo`
+
+link:https://github.com/projectatomic/skopeo[Skopeo] is an opensource command line tool maintained by the Project Atomic group. It works with images in remote repositories without the need to pull the images to the host. With `skopeo` the promotion process is much more streamlined:
```

Comment:
Split out open source into 2 words. Add a link to the Project Atomic homepage.

I would also emphasize skopeo as a tool that does not require privileged access to manipulate container images which is a contrast to docker

Reply:
When you say privileged access, do you mean that the tool doesn't require elevated access on the intermediate host or something else?